CONTRIBUTING.md

```diff
@@ -0,0 +1,227 @@
+# Contributing
+
+Thanks for considering contributing to Subspace Network!
+
+The best way to start is to join our [Forum](https://forum.subspace.network/) or
+[Discord](https://discord.gg/subspace-network), [official website](https://subspace.network/) has a bunch of learning
```

Comment:
should Slack be included?

Reply:
Certainly, we have already discussed this internally, we just need to have a process of external people to request access to engineering channel. The links Slack generates are temporary and don't work because of that. We'll probably have a form or something like that for people to put their email in so we can invite them individually.